cloud/filestore/libs/vfs_fuse/fs_impl_data.cpp

```diff
@@ -621,6 +697,14 @@ void TFileSystem::Flush(
             }
         };
 
+    if (Config->GetServerWriteBackCacheEnabled()) {
+        WriteBackCache->FlushData(fi->fh)
+            .Apply([](const auto&)
+                   { return MakeError(S_OK); })
+            .Subscribe(std::move(callback));
```

Comment:
what about other types of requests inside FSyncQueue there are also `Create` and `Fallocate` requests that are waiting there